js_arrays/task1.js

```diff
@@ -0,0 +1,21 @@
+const numbers = [2, -5, 0, 7, -3, 0, 10, -8];
+
+let positiveCount = 0;
+let negativeCount = 0;
+let zeroCount = 0;
+
+for (let i in numbers) {
```

Comment:
it can be an option but not the best one. `For in`  loop was mainly designed for objects.

Використовуючи цикл for, пройдіться по кожному елементу масиву numbers. - you should use regular `for` loop in this task.

Or if you can explain why does your solution work then I will accept it :)

Reply:
> Or if you can explain why does your solution work then I will accept it 🙂
@sttaran it works because it goes through the all elements in the array 🤷‍♂️